.travis.yml

```diff
@@ -6,12 +6,17 @@ node_js:
   - "10"
   - "11"
 
-sudo: false
-
 cache:
   yarn: true
 
 
+before_install:
+  - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
+  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+  - sudo apt-get update -qq
+  - sudo apt-get install -y -qq yarn
+  - yarn --version
```

Comment:
I would suggest to use the same steps as in all the other repos:
https://github.com/ember-cli/ember-cli/blob/453c105ef4a15feb3ed1e7c26dd81e835d15095a/tests/fixtures/app/yarn/.travis.yml#L21-L22

Reply:
yeah, I ended up doing that

Reply:
but its dumb that the thing https://yarnpkg.com/en/docs/install-ci says doesn't work 😠

Reply:
I totally agree. I'm also not sure why they recommend using `sudo` commands when there is clearly no need for it...